lms/static/sass/course/_info.scss

```diff
@@ -229,5 +229,13 @@ div.info-wrapper {
         }
       }
     }
+
+    @media print {
+      display: none;
+    }
+  }
+
+  @media print {
```

Comment:
See screenshot below:
![screen shot 2015-04-17 at 4 00 04 pm](https://cloud.githubusercontent.com/assets/6232546/7209900/fff57b1c-e51a-11e4-93e5-e7afa08701c3.png)
It's the same border for every other border you've asked for a screenshot for, just that it exists in different parts of the CSS.